soak_bench_scripts/soak.sh

```diff
@@ -38,4 +39,10 @@ rm target/release/integration_tests-*.d
 printf "Starting soak tests...\n"
 
 # Loop the test forever until it hits an error
-loop -q target/release/integration_tests-* |& tee -a -i -p --output-error=warn $LOGFILE
+os=$(uname)
+echo $os |& tee $LOGFILE
```

Comment:
This will overwrite an existing logfile - is that intended?  I don't see a problem, but just wanted to check since the behaviour has changed now.

Reply:
No, it's missing the `-a`. Will change.